packages/http-client-csharp/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@typespec/http-client-csharp",
+  "version": "0.2.0",
+  "author": "Microsoft Corporation",
+  "description": "The typespec library that can be used to generate C# models from a TypeSpec REST protocol binding",
+  "homepage": "https://github.com/Microsoft/typespec",
+  "readme": "https://github.com/Microsoft/typespec/blob/main/packages/http-client-csharp/readme.md",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/typespec.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/typespec/issues"
+  },
+  "keywords": [
+    "typespec"
+  ],
+  "type": "module",
+  "main": "dist/src/index.js",
+  "scripts": {
+    "clean": "rimraf ./dist ./temp",
+    "build": "tsc -p .",
+    "watch": "tsc -p . --watch",
+    "lint-typespec-library": "tsp compile . --warn-as-error --import @typespec/library-linter --no-emit",
+    "test": "mocha dist/test/**/*.js",
+    "test-official": "c8 mocha --forbid-only dist/test/**/*.js",
+    "lint": "eslint . --ext .ts --max-warnings=0",
+    "lint:fix": "eslint . --fix --ext .ts",
+    "format": "pnpm -w format:dir packages/http-client-csharp"
+  },
+  "files": [
+    "dist/src/**"
+  ],
+  "dependencies": {
+    "json-serialize-refs": "0.1.0-0",
+    "winston": "^3.8.2"
+  },
+  "peerDependencies": {
+    "@azure-tools/typespec-azure-core": ">=0.36.0 <1.0.0",
+    "@azure-tools/typespec-client-generator-core": ">=0.36.0 <1.0.0",
+    "@typespec/compiler": ">=0.50.0 <1.0.0",
+    "@typespec/http": ">=0.50.0 <1.0.0",
+    "@typespec/rest": ">=0.50.0 <1.0.0",
+    "@typespec/versioning": ">=0.50.0 <1.0.0",
+    "@typespec/openapi": ">=0.50.0 <1.0.0"
+  },
+  "devDependencies": {
+    "@azure-tools/typespec-azure-core": "0.40.0",
+    "@azure-tools/typespec-client-generator-core": "0.40.0",
+    "@typespec/compiler": "0.54.0",
```

Comment:
Is the idea to update these as part of each release, or to update these ad hoc?

Reply:
We typically update them after each release.  There is no guarantee on the SLA but is within a day or two usually.